### PR TITLE
chore(migrations-irec): added irec beneficiaries to seed file

### PR DESCRIPTION
**Pull request by ioncreature**

### CODE REVIEW

File: packages/tools/migrations-irec/config/seed.sql [hunk 1]
```diff
@@ -664,4 +664,75 @@ VALUES (
         'd49f7fed-2f79-4d10-8985-41a0c3e9ba03',
         'METER003',
         'Biomass Facility description'
-    );
\ No newline at end of file
+    );
+
+INSERT INTO public."platform_organization" (
```
Comment (soanvig) [inline packages/tools/migrations-irec/config/seed.sql]: it's minor but maybe https://www.postgresqltutorial.com/postgresql-insert-multiple-rows/
would be more readable?

File: packages/tools/migrations-irec/config/seed.sql [hunk 2]
```diff
@@ -664,4 +664,75 @@ VALUES (
...
+    "createdAt",
+    "updatedAt",
+    id,
+    irecBeneficiaryId,
+    organizationId
+)
+VALUES (
+           '2020-03-30 09:55:25.962333+02',
+           '2020-03-30 09:55:25.962333+02',
+           1,
```
Comment (soanvig) [inline packages/tools/migrations-irec/config/seed.sql:679]: Inserting that manually doesn't increment serial field, if this should be autoincrementing, which will cause conflicts if you want to insert anything later on. is this autoincrement field?

**Review by soanvig — CHANGES_REQUESTED** [review]

File: packages/tools/migrations-irec/config/seed.sql (hunk 2 above)
Comment (ioncreature) [reply packages/tools/migrations-irec/config/seed.sql:679]: fixed by adding `nextval`

File: packages/tools/migrations-irec/config/seed.sql (hunk 1 above)
Comment (ioncreature) [reply packages/tools/migrations-irec/config/seed.sql]: fixed

**Review by soanvig — APPROVED** [review]

**Review by josipbagaric — APPROVED** [review]

**Pull request merged**